sig-bug-triage-process.md

```diff
@@ -0,0 +1,54 @@
+## SIG Bug Triage Process
+
+SIGs should be triaging the issues with their matching label during every SIG
+meeting. You can view
+[a list of recently filed issues](https://github.com/spinnaker/spinnaker/labels/sig%2Fplatform)
+on GitHub to and click through the ones filed since your last meeting. (That
+link shows issues for the Platform SIG, but you can easily edit the filter from
+there.)
+
+Each issue should have one of three outcomes:
+
+1.  **Assigned to a user or added to an actively-maintained GitHub project.** An
+    actively-maintained project has a team a people who are using it to plan
+    future work. If you assign it to a project, you should also add the
+    `no-lifecycle` tag so that Spinnakerbot doesn't automatically close it.
```

Comment:
I'm hesitant to add `no-lifecycle` to all these issues.

I'd obviously rather have the question be moot, by having all issues that are assigned to a person or project be resolved within 90-day autoclose window.  But in the case that something gets triaged/assigned and has no updates for 90 days, it sort of does seem reasonable to auto-close it (which at least will send out notifications that the issue is stale) vs just having it be part of an ever-increasing backlog.

Reply:
+1 hesitant, agree it's good for cleanup.

If we want to lean into the project-based approach, we could have a new workflow that would pull the issue back out of the project / un-assign it and then it'd be available for triage again. At that point, it could be a human discussion of if whether it'd be worth keeping open or not.

Reply:
Okay, I removed that sentence.